### PR TITLE
Wip/bhahn/BSPIMX8M-3195: include chapters from imx-common in imx8 manuals

### PR DESCRIPTION
**Pull request by BHahn42618**
include the following chapters from imx-common:
- source/bsp/imx-common/development/format_sd-card.rsti
- source/bsp/imx-common/development/uuu.rsti
- source/bsp/imx-common/development/netboot.rsti
- source/bsp/imx-common/peripherals/i2c-bus.rsti
- source/bsp/imx-common/peripherals/network.rsti
- source/bsp/imx-common/peripherals/pin-muxing.rsti
- source/bsp/imx-common/emmc.rsti

Generalize source/bsp/imx9/peripherals/display.rsti move to imx-common and include in imx8 and imx9 @cstoidner 

Do not include bsp/imx-common/development/standalone_build.rsti, because bigger changes are needed before this can be included in imx8.

### CODE REVIEW

File: source/bsp/imx8/imx8mm/head.rst [hunk 1]
```diff
@@ -73,6 +73,7 @@
 .. |ref-network| replace:: :ref:`Network Environment Customization <imx8mm-head-network>`
 .. |ref-setup-network-host| replace:: :ref:`Setup Network Host <imx8mm-head-development>`
 .. |ref-usb-otg| replace:: :ref:`X2 <imx8mm-head-components>`
+.. |ref-disable-emmc-part| replace:: :ref:`Disable booting from eMMC boot partitions <emmc-disable-boot-part>`
```
Comment (ymoog) [inline source/bsp/imx8/imx8mm/head.rst:77]: This substitution is used in the uuu chapter and has nothing to do with the emmc.rsti you (want) to modify here. Maybe happend accidentally when rebasing? You did not introduce an u-boot-offset as you said you do in your commit message.

File: source/bsp/imx8/imx8mm/head.rst [hunk 2]
```diff
@@ -78,7 +78,7 @@
 
 
 .. IMX8(MM) specific replacements
-.. |pollux-sbc-network| replace:: \
+.. |sbc-network| replace:: \
```
Comment (ymoog) [inline source/bsp/imx8/imx8mm/head.rst:81]: Why can you change that all of a sudden? Please provide an explanation in your commit message.

**Review by ymoog — CHANGES_REQUESTED** [review]

File: source/bsp/imx8/imx8mm/head.rst (hunk 1 above)
Comment (BHahn42618) [reply source/bsp/imx8/imx8mm/head.rst:77]: The `emmc-disable-boot-part` marker is introduced in emmc.rsti, so I created the substitution for this marker here, but you are correct, the substitution is used in uuu.rsti the first time (which gets included in the next commit). If you want I can move the Introduction of the substitution to the uuu commit.

I never said, that I would introduce an u-boot-offset substitution. I said I would use it and I did just that. I replaced the hardcoded 32kiB in emmc.rsti with the u-boot-offset substitution. The substitution already exists in our manual there is no need to introduce it. I can just use it.

File: source/bsp/imx8/imx8mm/head.rst (hunk 2 above)
Comment (BHahn42618) [reply source/bsp/imx8/imx8mm/head.rst:81]: Done. Hope this is okay now.

File: source/bsp/imx8/imx8mm/head.rst (hunk 1 above)
Comment (ymoog) [reply source/bsp/imx8/imx8mm/head.rst:77]: > The `emmc-disable-boot-part` marker is introduced in emmc.rsti, so I created the substitution for this marker here, but you are correct, the substitution is used in uuu.rsti the first time (which gets included in the next commit). If you want I can move the Introduction of the substitution to the uuu commit.

Yes, please.

> 
> I never said, that I would introduce an `|u-boot-offset|` substitution. I said I would use it and I did just that. I replaced the hardcoded 32kiB in emmc.rsti with the u-boot-offset substitution. The substitution already exists in our manual there is no need to introduce it. I can just use it.

Ah, makes sense. Thanks.

Comment (BHahn42618) [reply source/bsp/imx8/imx8mm/head.rst:77]: Ok moved the introduction of the emmc-disable-boot-part marker to the uuu commit.

**Review by ymoog — APPROVED** [review]

**Review by tremmet — APPROVED** [review]

**Comment by jonas-rem** [comment]
@BHahn42618 is this PR ready for merging? If so, you can set it to "Ready for review" and I will merge it later.

**Pull request merged**